### PR TITLE
PVR API 5.8.0 changes

### PR DESCRIPTION
**Pull request by fuzzard**
API changes as per https://github.com/xbmc/xbmc/pull/13228

Merge after thats been merged into Kodi. Will monitor incase any further api changes done.
Also some minor cleanups, categorising unused API's, and added missing unused stubs for completeness

### CODE REVIEW

**Comment by fuzzard** [comment]
@ksooo is the GetStreamTimes implementation ok?

Its livetv, so ptsBegin and ptsEnd can be just 0, as per the 5.8.0 api commit. Not entirely sure about ptsStart, and as we dont have a need to keep track of start time, is it acceptable to just return current time for startTime or should i be throwing in a startTime that is set when GetLiveStreamProperties is called for the start of a stream?

**Comment by ksooo** [comment]
Please ask @FernetMenta. He created that API function and knows best.

**Comment by ksooo** [comment]
I think you don't need to implement this at all, because your add-on does neither implement a debugger nor own streaming. You're using Kodi player to playback the urls you provide, right MB

**Comment by fuzzard** [comment]
I emailed fernet and asked about this. After hearing his reply, I'll remove the i implementation of getstreamtimes and just handle any further changes as they occur.
 Thanks for all your work you have done with the PVR interface. I'll push the appropriate change tomorrow

File: src/HDHomeRunTuners.cpp [hunk 1]
```diff
@@ -400,6 +400,7 @@ PVR_ERROR HDHomeRunTuners::PvrGetChannelGroupMembers(ADDON_HANDLE handle, const
       channelGroupMember.strGroupName[sizeof(channelGroupMember.strGroupName) - 1] = '\0';
       channelGroupMember.iChannelUniqueId = jsonChannel["_UID"].asUInt();
       channelGroupMember.iChannelNumber = jsonChannel["_ChannelNumber"].asUInt();
+      channelGroupMember.iSubChanneNumber = jsonChannel["_SubChannelNumber"].asUInt();
```
Comment (MilhouseVH) [inline src/HDHomeRunTuners.cpp]: Typo here (`.iSubChanneNumber =` - missing `l`) - I'll push a PR shortly.

Comment (fuzzard) [reply src/HDHomeRunTuners.cpp]: yeah sorry, my bad on that one. just noticed as i tried to rebase some other PR's

**Pull request merged**